.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Issue
+
+https://github.com/SAP/open-ux-tools/issues/ _FILL IN ISSUE NUMBER HERE_
+
+## Description
+
+_FILL IN DETAILS HERE_
+
+## Checklist for Pull Requests
+
+- [ ] Supplied as many details as possible on this change.
+- [ ] Included the link to the associated issue in the [issues](https://github.com/SAP/open-ux-tools/issues)
+- [ ] The code is **easy to read and maintainable** by others.
+- [ ] The code conforms to the general development principles.
+- [ ] Functions and variables are suitably named and are sufficiently descriptive.
+- [ ] The code has **unit tests** where applicable and is easily unit-testable.
+- [ ] This PR is associated with a discrete task (smaller discrete tasks can be more efficiently reviewed).
+- [ ] This branch is appropriately named for the associated issue.
```

Comment:
Couldn't find information about appropriate naming of branches. If we aim for branch names like:

```
fix|feat|docs|style|refactor|perf|test|chore|revert|WIP/<ISSUE_NUMBER>/<SHORT-DESCRIPTIO>
```
e.g.

```
fix/123/handle-exception
```

we should document it, perhaps in https://github.com/SAP/open-ux-tools/blob/main/docs/Guidelines.md
What do you think?

Reply:
yes. this is now irrelevant as a checklist if we are providing link to dev guidelines. 
And this info belongs in the guidelines.